test/chpldoc/compflags/version/projectVersion.doc.chpldocopts

```diff
@@ -0,0 +1,7 @@
+--project-version 21.0.3 # projectVersion.no-tag.doc.good
+--project-version 21.0.3-alpha   # projectVersion.tag.doc.good
```

Comment:
It'd be nice to have a few more success modes covered by testing.  I'd like to see:
- all the supported ending tags
- a version with all single digits (e.g. `1.2.3` or whatever)
- putting the multi-digits at various places around the `.` (e.g. also covering `0.21.3`, for instance)